be/src/common/config.h

```diff
@@ -1198,6 +1198,7 @@ DECLARE_mInt32(group_commit_queue_mem_limit);
 // group_commit_wal_max_disk_limit=1024 or group_commit_wal_max_disk_limit=10% can be automatically identified.
 DECLARE_mString(group_commit_wal_max_disk_limit);
 DECLARE_Bool(group_commit_wait_replay_wal_finish);
+DECLARE_Int32(group_commit_wal_version);
```

Comment:
should not be a config, because user may change it

Reply:
already change the code,take a look please @mymeiyi